mungegithub/mungers/cherrypick-auto-approve.go

```diff
@@ -138,9 +145,10 @@ func (c *CherrypickAutoApprove) Munge(obj *github.MungeObject) {
 		// If the parent was for milestone v1.2 but this PR has
 		// comments saying it was 'on branch release-1.1' we should
 		// not auto approve
-		parentMilestone := parent.ReleaseMilestone()
-		if parentMilestone != branchImpliedMilestone {
-			glog.Errorf("%d: parentReleaseMilestone=%q but branch is %q", *obj.Issue.Number, parentMilestone, obj.Branch())
+		parentMilestone, ok := parent.ReleaseMilestone()
+		if parentMilestone != branchImpliedMilestone || !ok {
+			branch, _ := obj.Branch()
```

Comment:
why do I have to do this again? Didn't we get branch in https://github.com/kubernetes/contrib/pull/1357/files#diff-b60bc1c6cc564c1b15eacccaca226ce6R126